src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/MinAfFilter.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect.filtering;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.commons.lang.mutable.MutableBoolean;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+
+import java.util.*;
+
+public class MinAfFilter extends HardFilter {
+    private final double minAf;
+
+    public MinAfFilter(final double minAf) {
+        this.minAf = minAf;
+    }
+
+    @Override
+    public ErrorType errorType() { return ErrorType.ARTIFACT; }
+
+    @Override
+    public boolean isArtifact(final VariantContext vc, final Mutect2FilteringEngine filteringEngine) {
+        MutableBoolean b = new MutableBoolean(false);
```

Comment:
This could be cast as an `anyMatch`:

```
return vc.getGenotypes().stream().filter(filteringEngine::isTumor)
                .filter(g -> g.hasExtendedAttribute(GATKVCFConstants.ALLELE_FRACTION_KEY))
                .anyMatch(g -> {
                    final double[] alleleFractions = GATKProtectedVariantContextUtils.getAttributeAsDoubleArray(g, GATKVCFConstants.ALLELE_FRACTION_KEY, () -> null, 1.0);
                    double max = alleleFractions[0];
                    for (int i=0; i<alleleFractions.length; i++) {
                        //Only include allele fractions of non symbolic alleles for GVCFs
                        if (!(vc.hasSymbolicAlleles() && i == alleleFractions.length - 1)) {
                            max = alleleFractions[i] > max ? alleleFractions[i] : max;
                        }
                    }
                    return max < minAf;
                });
```